packages/sampler/stories/default/VirtualGridList.js

```diff
@@ -108,20 +112,22 @@ storiesOf('Moonstone', module)
 		'VirtualList.VirtualGridList',
 		() => (
 			<VirtualGridList
-				dataSize={updateDataSize(number('dataSize', Config, defaultDataSize))}
-				direction={select('direction', prop.direction, Config, 'vertical')}
-				focusableScrollbar={boolean('focusableScrollbar', Config)}
-				horizontalScrollbar={select('horizontalScrollbar', ['auto', 'hidden', 'visible'], Config, 'auto')}
+				dataSize={updateDataSize(number('dataSize', VirtualGridListConfig, defaultDataSize))}
+				direction={select('direction', prop.direction, VirtualGridListConfig)}
+				focusableScrollbar={boolean('focusableScrollbar', VirtualGridListConfig)}
+				horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, VirtualGridListConfig)}
 				itemRenderer={renderItem}
 				itemSize={{
-					minWidth: ri.scale(number('minWidth', Config, 180)),
-					minHeight: ri.scale(number('minHeight', Config, 270))
+					minWidth: ri.scale(number('minWidth', VirtualGridListConfig, 180)),
+					minHeight: ri.scale(number('minHeight', VirtualGridListConfig, 270))
 				}}
+				noScrollByWheel={boolean('noScrollByWheel', VirtualGridListConfig)}
 				onScrollStart={action('onScrollStart')}
 				onScrollStop={action('onScrollStop')}
-				spacing={ri.scale(number('spacing', Config, 20))}
-				verticalScrollbar={select('verticalScrollbar', ['auto', 'hidden', 'visible'], Config, 'auto')}
-				wrap={wrapOption[select('wrap', ['false', 'true', '"noAnimation"'], Config)]}
+				spacing={ri.scale(number('spacing', VirtualGridListConfig, 20))}
+				spotlightDisabled={boolean('spotlightDisabled', VirtualGridListConfig, false)}
+				verticalScrollbar={select('verticalScrollbar', prop.scrollbarOption, VirtualGridListConfig)}
+				wrap={wrapOption[select('wrap', ['false', 'true', '"noAnimation"'], VirtualGridListConfig)]}
```

Comment:
Why don't you define `prop = {wrapOption: ['false', 'true', '"noAnimation"'], ...}` and use `prop.wrapOption` instead of `['false', 'true', '"noAnimation"']`?

Reply:
The properties of variable `prop` are mostly used for presenting second parameter `items` of APIs provided in addon-knobs. Whereas `wrapOption` looks like to be used foronly  type conversion. That's the reason I did not put 'wrapOption' inside 'prop'.